packages/console/src/components/Table/TableError.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import ErrorImage from '@/assets/images/table-error.svg';
+
+import Button from '../Button';
+import * as styles from './TableError.module.scss';
+
+type Props = {
+  title?: string;
+  content?: string;
+  onTryAgain?: () => void;
+};
+
+const TableError = ({ title, content, onTryAgain }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <tr>
+      <td colSpan={0}>
```

Comment:
Is `colSpan=0` still working on major browsers?
https://stackoverflow.com/a/52355253/3431443

Reply:
It works fine in Chrome and Safari, and is more elegent then 1000.

Reply:
OK, but how about firefox? Still feeling unsafe using it, since the W3 HTML 5.0 standard clearly stated that colspan value must be a valid non-negative integer greater than zero.
So... how about provide a prop to allow setting the column count of the <TableError> component?

Reply:
Providing a prop is not a good idea, if 0 is not safe, then we can set 1000.

Reply:
If we use `colSpan=1000`, then if we use `table-layout: fixed`, it might just be broken.

Reply:
You are right, both `colSpan=0` and `colSpan=1000` are not perfect, our "empty state" component will have this problem too, let's move forward and refactor it later after researching? Created LOG-1974

Reply:
ok